srcpkgs/azote/template

```diff
@@ -1,24 +1,23 @@
 # Template file for 'azote'
 pkgname=azote
-version=1.7.10
-revision=2
+version=1.8.1
+revision=1
 build_style=python3-module
 hostmakedepends="python3-setuptools"
-depends="python3>=3.4 python3-setuptools python3-gobject python3-Pillow gtk+3 feh python3-send2trash xrandr grim slurp ImageMagick python3-yaml"
+depends="python3>=3.4 python3-gobject python3-Pillow gtk+3"
```

Comment:
Please drop `>=3.4`, it's too old to work with like anything.

Reply:
This is embarrassing, totally overlooked the symlink. Fixed now.

`python3` is replaced with `python3-setuptools`